manifests/init.pp

```diff
@@ -226,6 +226,11 @@
   $slurmd_user            = 'root',
   $slurmd_user_group      = 'root',
 
+  # Munge key
+  Boolean $manage_munge                 = false,
```

Comment:
These 3 new parameters will need documentation at the top of the file otherwise the puppet-lint tests will fail.